packages/account/src/Configs/financial-details-config.tsx

```diff
@@ -1,6 +1,7 @@
-import { localize } from '@deriv/translations';
```

Comment:
This file is also not a component, it's fine to keep it as a ts file

Reply:
Same here, we use React types